tool/jt.rb

```diff
@@ -709,42 +709,84 @@ def compile_cext(name, ext_dir, target, *clang_opts)
     end
   end
 
-  def pr(*args)
-    out, _err   = sh 'git', 'remote', capture: true
-    remotes     = out.split
-    remote_urls = remotes.map do |remote|
-      out, _err = sh 'git', 'config', '--get', "remote.#{remote}.url", capture: true
-      [remote, out.chomp!]
+  module PR
+    include ShellUtils
+
+    def pr_clear
+      bb       = bb(remote_urls)
+      data     = Net::HTTP.get(URI('https://api.github.com/repos/graalvm/truffleruby/pulls'))
+      prs_data = JSON.parse data
+      open_prs = Set.new prs_data.map { |prd| prd.fetch 'number' }
```

Comment:
Why not just `.uniq` here?
`Set` feels like it could reorder elements which would be rather surprising (also it's an extra dependency).